### ISSUE TITLE
first fits file

### ISSUE THREAD

**Issue by feltre** [feltre/FITS_files_creation#2]
You can find in "data" the first fit files (with 51 emission lines) in the format we agreed on. 
This file contains information of AGN models with fix xid, nh and alpha for any value of Z and U.
If I have understood correctly, I know have to make other files like this one for all the values of xid, nh and alpha. Correct?
Are we merging them in a final single file or not?
@jacopo-chevallard

**Comment by jacopo-chevallard** [comment]
Excellent! Two small things: 

- in the "wavelengths" extension the line wavelengths are not printed (i.e., they are all set to 0, but I guess you are aware of this);
- what did you choose fort the line labels (i.e. the names in the FITS columns)? Whatever you choose avoid mixing single (_) and double (__) underscores, as this will create confusion.

About creating the other files: the idea was to have a different file for each combination of xid, nh and alpha, i.e. **not** merging them in a single file. The main reason was because Julia's FITS files will be substantially larger than yours, and it is not practical to handle files of GB size. 
However, having the different xid in the same file means that a single file contains the "basic", ready-to-use model, and this is certainly an advantage. 
Julia's files, with the same format of the one you created, should occupy ~150 MB (but this should be tested), so adding xid in the same file means going to ~500 MB per file. If this is true, then I would keep the different xid in different files, otherwise, if the size if smaller, we can put them together in a single one.   

In any case, I would keep the file format as similar as possible among yours and Julia's models.

What do you think?

**Comment by feltre** [comment]
Thanks Jacopo!
Now I should have fixed you two points (see new table_NLR.fits).
If you spot something else, please, let me know and I will fix it! 
The new line labels are in the format ELEMENT_IONIZATION@WAVELENGTH (e.g. O_3@5007) as discussed (will be happy to change it if there are other preferences).

I also agree in not merging the files. And we will keep Julia's models as close as possible to these ones (i.e. Julia in addition will have different ages). I will ask her a couple of things tomorrow before finalizing everything!

**Comment by jacopo-chevallard** [comment]
Great, now it looks perfect! :-)

We do not have any line at lambda > 6731? is this made on purpose? (i guess there are some strong lines at longer wavelengths too...)

**Comment by feltre** [comment]
We do have several lines at longer wavelengths (up to IR at least) but for the moment I have considered only those lines in output in the SF models (but Pa_alpha because pyCloudy has a problem in reading it, but I was planning to fix this soon).
I can discuss next week with Stephane and Julia to see how to extend this line list and select at least the most important ones at lambda > 6731.

**Comment by scharlot** [comment]
Hi Jacopo,

I think this is because we decided to stick to the UV/optical lines we control best for the moment...

**Comment by JuliaGutkin** [comment]
Ok, we should discuss about these lines next week to see how to implement them (but I think we will have to run again the entire grid to add new lines)

**Comment by scharlot** [comment]
There is no rush in adding these lines -- we can talk with Alba and Anna next week.

**Comment by feltre** [comment]
Julia, no worries because BANGS/BEAGLE reads only the .ineb files in which you have already in output more then 200 lines (so I do not expect many lines to be missing). We can keep track of some important ones that might be missing while testing the code and then run the grid again later on.

**Comment by jacopo-chevallard** [comment]
Right, in the "full list" I have from Julia (with > 200 lines) there are many at lambda > 6731, so I thought it  was only a matter of selecting them. But I agree, no rush, once the system is in place it should be easy to add further lines.
